package-lock.json

```diff
@@ -60,6 +61,10 @@
       },
       "engines": {
         "node": "^14.18 || ^16.13 || >=18"
+      },
+      "optionalDependencies": {
+        "@adobe/adobe-intl-aladdin": "4.3.8-beta.5",
+        "@adobe/adobe-intl-harvester": "3.2.8"
       }
```

Comment:
@vtsaplin Nice addition with optional dependencies. I was just wondering if there is still a possibility that might break CI/CD even with the property of `"optional": true` in private Artifactory packages?

Reply:
> I was just wondering if there is still a possibility that might break CI/CD even with the property of "optional": true in private Artifactory packages?

Looks like it's all good—the actions in this PR managed to install dependencies without any issues.